tmt/steps/provision/__init__.py

```diff
@@ -1477,6 +1483,59 @@ def essential_requires(cls) -> list['tmt.base.Dependency']:
 
         return []
 
+    def acquire_log(self, log_name: str) -> Optional[str]:
```

Comment:
Nomenclature comment: in tmt code the form of `[a-z]+name` is more common than `[a-z]+_name` - `filename`, `guestname, also `filepath`, etc. - therefore I would recommend using `logname` instead of `log_name`.